src/connectors/tagging/tagging.js

```diff
@@ -1,264 +1,130 @@
-    const tagsArray = storedTags || []
-    const filteredStoredTags = tagsArray.filter(
-      item => usedTags.indexOf(item) < 0
-    )
-    return value ? matchSorter(filteredStoredTags, value) : []
-  }
-
-  /* Render Component
-    –––––––––––––––––––––––––––––––––––––––––––––––––– */
   return (
-    <TaggingWrapper>
-      <Downshift onSelect={onSelect}>
-        {({
-          getInputProps,
-          getItemProps,
-          isOpen,
-          highlightedIndex
-        }) => (
-          <div>
-            <TaggingWell onMouseUp={onMouseUp}>
-              {placeholder && !hasTags() && (
-                <TaggingPlaceholder>
-                  {localize('tagging', 'add_tags')}
-                </TaggingPlaceholder>
-              )}
-
-              {!!hasTags() && (
-                <Chips
-                  tags={tags.used}
-                  marked={tags.marked}
-                  toggleActive={toggleActive}
-                  removeTag={removeTagAction}
-                />
-              )}
-
-              <Taginput
-                highlightedIndex={highlightedIndex}
-                getInputProps={getInputProps}
-                hasTags={!!hasTags()}
-                inputRef={inputRef}
-                value={inputvalue}
-                focused={inputFocused}
-                setValue={setInputValue}
-                setFocus={setFocus}
-                setBlur={setBlur}
-                closePanel={closePanel}
-                addTag={addTagAction}
-                handleRemoveAction={handleRemoveAction}
-                makeTagsInactive={makeTagsInactive}
-                storedTags={storedTagsList()}
-              />
-            </TaggingWell>
-
-            {!isOpen || !storedTagsList().length ? null : (
-              <TaggingTypeaheadWrapper>
-                <TaggingTypeaheadList>
-                  {storedTagsList().map((item, index) => {
-                    return (
-                      <TaggingTypeaheadItem
-                        active={highlightedIndex === index}
-                        key={`item-${index}`}
-                        {...getItemProps({
-                          item,
-                          index
-                        })}>
-                        {item}
-                      </TaggingTypeaheadItem>
-                    )
-                  })}
-                </TaggingTypeaheadList>
-              </TaggingTypeaheadWrapper>
-            )}
-          </div>
-        )}
-      </Downshift>
-
-      {tags && tags.suggested && (
-        <Suggestions
-          tags={tags}
-          suggestions={tags.suggested}
-          addTag={addTagAction}
-        />
-      )}
-    </TaggingWrapper>
+    <Tagging
+      tags={tagsObject}
+      storedTags={storedTags}
+      addTag={addTag}
+      activateTag={activateTag}
+      deactivateTag={deactivateTag}
+      deactivateTags={deactivateTags}
+      removeTag={removeTag}
+      removeTags={removeTags}
+      closePanel={closePanel}
+    />
```

Comment:
This is now a straight connector which loads the `Tagging` component. This houses the state management functionality. This diff is useless.